python/lsst/qa/explorer/writeParquet.py

```diff
@@ -0,0 +1,107 @@
+from lsst.daf.persistence.butler import Butler
+from lsst.pex.config import (Config, Field, ConfigField, ListField, DictField, ConfigDictField,
+                             ConfigurableField)
+from lsst.pipe.base import Task, CmdLineTask, ArgumentParser, TaskRunner, TaskError
+from lsst.coadd.utils import TractDataIdContainer
+from lsst.pipe.tasks.multiBand import MergeSourcesTask, MergeSourcesConfig
+from lsst.pipe.tasks.multiBand import _makeGetSchemaCatalogs
+from lsst.coadd.utils.coaddDataIdContainer import ExistingCoaddDataIdContainer
+
+
+import functools
+import re
+import pandas as pd
+
+from .table import ParquetTable
+
+
+class WriteObjectTableConfig(MergeSourcesConfig):
+    priorityList = ListField(dtype=str, default=['HSC-G', 'HSC-R', 'HSC-I', 'HSC-Z', 'HSC-Y'],
+                             doc="Priority-ordered list of bands for the merge.")    
+    engine = Field(dtype=str, default="pyarrow", doc="Parquet engine for writing (pyarrow or fastparquet)")
+
+class WriteObjectTableTask(MergeSourcesTask):
+    """Write filter-merged source tables to parquet
+    """
+    _DefaultName = "writeObjectTable"
+    ConfigClass = WriteObjectTableConfig
+
+    inputDatasets = ('forced_src', 'meas', 'ref')
+    outputDataset = 'obj'
+
+    # Hack, since we're not using this...
+    getSchemaCatalogs = lambda x : {}
```

Comment:
What's the hack? If you're overriding something that you want to do nothing, you should do something like this:

```
def getSchemaCatalogs(self, x):
    """Overriding the default to do nothing."""
    pass
```

Reply:
The hack is that this function needs to return a dictionary.  I can write a proper method rather than a lambda, I suppose.

Reply:
Yes, please write a function.

Reply:
(done already)